packages/usa-tooltip/src/styles/_usa-tooltip.scss

```diff
@@ -28,7 +28,6 @@ $triangle-size: 5px;
   font-size: size("ui", $theme-tooltip-font-size);
   opacity: 0; // Required for recalculating position.
   padding: units(1);
-  pointer-events: none;
```

Comment:
> [!Note]
> We needed to remove the `pointer-event` styles here because they were causing the tooltip to be removed from the `usa-tooltip` hit boundary. This caused `mouseleave` to be triggered when hovering over the tooltip.